src/components/MultiSelect/MultiSelect.stories.mdx

```diff
@@ -72,23 +75,7 @@ import * as stories from "./MultiSelect.stories.tsx";
 
 ## Component Props
 
-<ArgsTable story="MultiSelect ListBox" />
-
-## Accessibility
-
-The `MultiSelect` variant `listbox` leverages `downshift.js` to assist with accessibility compliance.
-By using Downshift's `useSelect` hook and prop getters `getToggleButtonProps`, `getMenuProps`, `getItemProps` we are able to create a W3 WAI-ARIA compliant listbox that behaves like a select element. Up and down arrow keys are used for navigating through the `MultiSelect` options.
-
-The MultiSelect variant `dialog` looks similar to the Listbox variant, but because of its “clear” and “apply” buttons, it has some key functionality differences, and accessibility must be handled differently. When the multiselect is open, focus is trapped inside the dropdown menu, and the tab key is used to move through the checkboxes and buttons inside the dialog modal.
-
-Resources:
-
-- [downshift.js documentation](https://www.downshift-js.com/)
-- [Reservoir Checkbox Component](http://localhost:6006/?path=/docs/components-form-elements-checkbox--checkbox#accessibility)
-- [W3C WAI Listbox Example](https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html)
-- [W3C WAI on using ARIA role=dialog](https://www.w3.org/WAI/GL/wiki/Using_ARIA_role%3Ddialog_to_implement_a_modal_dialog_box)
-
-## MultiSelect Listbox Variant
+### MultiSelect Listbox Variant
```

Comment:
This is now a level 3 heading and under the `Component Props` heading, but the `MultiSelect Dialog Variant` is still a level 2 heading after the `Accessibility` heading. So, this should be updated as well as the table of contents.

Reply:
Yeah, this component is weird with how the vareiants are broken out.  To make it easier to digest, I will rearrange a few things and add to the descriptions.